inst/roxygen2-tags.yml

```diff
@@ -1,413 +1,465 @@
 - name: aliases
   description: >
-    Add additional aliases to the topic. Use `NULL` to suppress
-    the default alias automatically generated by roxygen2.
+    <p>Add additional aliases to the topic. Use <code>NULL</code> to suppress
```

Comment:
unless we wanted to keep the md versions, in that case maybe we could store the converted to html somewhere else ?

Reply:
I think using HTML directly is good enough.